WatchFaceFormat/Weather/watchface/src/main/res/raw/watchface.xml

```diff
@@ -74,15 +74,11 @@
                 </Group>
                 <Group height="47" name="CurrentConditionIcon" width="47" x="215" y="0">
                   <Condition>
-                    <!--
-                      When using CONDITION, it is a requirement that all
```

Comment:
What did this mean? And what was it referencing? Sounds scary.

Reply:
That would have been painful.  Why can't I have a WF that only shows rain, and is normal for other things.

Reply:
You can 😀